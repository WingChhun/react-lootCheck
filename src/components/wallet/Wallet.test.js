import React from 'react';
import {shallow} from 'enzyme';
import Wallet from "./wallet.container";

const props = {};

describe(`Component - Wallet`, () => {
    const app = shallow(<Wallet {...props}/>);

    it(`renders without crashing`, () => {
        expect(Wallet).toMatchSnapshot();
    })

})
