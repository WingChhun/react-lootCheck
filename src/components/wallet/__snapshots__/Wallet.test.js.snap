// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - Wallet renders without crashing 1`] = `[Function]`;
